cmd/servicelog/post.go

```diff
@@ -87,14 +89,36 @@ func (o *PostCmdOptions) Validate() error {
 	return nil
 }
 
+func (o *PostCmdOptions) CheckServiceLogsLastHour() bool {
+	getAllMessages := false      // we need just manual entries
+	getInternalLogsOnly := false // we need all messages
+	numberOfHours := 1           // number of hours we need to wait for svc logs
+	timeStampToCompare := time.Now().Add(-time.Hour * time.Duration(numberOfHours))
+	serviceLogs, err := GetServiceLogsSince(o.ClusterId, timeStampToCompare, getAllMessages, getInternalLogsOnly)
+	if err != nil {
+		log.Fatalf("failed to fetch service logs: %q", err)
+	}
+	if len(serviceLogs) > 0 {
+		for _, svclog := range serviceLogs {
+			fmt.Println("Below service Log has been subitted in last 60 minutes\nDescription: ", svclog.Description())
+		}
+		return true
+	}
+	return false
+}
+
 func (o *PostCmdOptions) Run() error {
 	if err := o.Init(); err != nil {
 		return err
 	}
 	if err := o.Validate(); err != nil {
 		return err
 	}
-
+	if term.IsTerminal(int(os.Stdout.Fd())) && o.CheckServiceLogsLastHour() {
```

Comment:
Had a talk today about another card that would add a check like this.
It might be useful to still print that there was a warning even if it doesn't stop the script (e.g. to STDERR) just so it can be seen.